contracts/starknet/authenticator/authenticator.cairo

```diff
@@ -0,0 +1,15 @@
+%lang starknet
+from starkware.starknet.common.syscalls import call_contract
+
+# Forwards `data` to `target` without verifying anything.
+# TODO: use ADDRESS instead of felt
+@external
+func execute{syscall_ptr: felt*, range_check_ptr}(to: felt, selector: felt, calldata_len: felt, calldata: felt*):
+    # TODO: Actually verify the signature
+
+    # Call the space contract
```

Comment:
We should not mention "space" in this contract, this contract will be general purpose, it could be used to call other kind of contract.